docs/api/List.md

```diff
@@ -1 +1,156 @@
-# TODO
+# List Module
```

Comment:
put spaces after the commas in the lists in these examples so they're more readable